playwright/tests/uniqueLink.spec.js

```diff
@@ -0,0 +1,37 @@
+const { test, expect} = require('../fixtures/pageFixtures');
+
+test.beforeEach('Navigation to studies', async ({loginPage, studiesPage}) => {
+    await loginPage.goto();
+    await loginPage.login();
+    await studiesPage.goto();
+});
+
+test('Unique link test', async ({studiesPage, editTreeTestPage, previewPage}) => {
+    //Sets up unique link
+    await studiesPage.clickEditButtonForFirstTableRow();
+    await expect(editTreeTestPage.header).toContainText('Edit Tree Test');
+    await editTreeTestPage.acceptResponses.click();
+    await editTreeTestPage.submitButton.click();
+    await studiesPage.clickEditButtonForFirstTableRow();
+    await editTreeTestPage.uniqueParticipantLink.click();
+    await editTreeTestPage.addNameForUniqueParticipant.clear();
+    await editTreeTestPage.addNameForUniqueParticipant.fill('Name');
+    await editTreeTestPage.addUniqueParticipantButton.click();
+    await editTreeTestPage.copyUniqueLink('Name');
+
+    //Taking the unique URL from the page, and passing it to the preview page
+    var uniqueUrl = await editTreeTestPage.uniqueLink;
+    await previewPage.goto(uniqueUrl);
```

Comment:
Can we change the name of the page to view instead so that it is consistent with the actual page name, Also preview suggests that the page is used only for preview which is not the case.

Reply:
Given we also have tests that utilize the same elements but in the preview setting, Renaming this View would then not make sense to those. I'll have a think and try to figure out a name that makes sense to both functionalities